### PR TITLE
sqlmap 1.5.5

### PR DESCRIPTION
**Pull request by alebcay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
```
==> FAILED
sqlmap:
  * The homepage URL http://sqlmap.org should use HTTPS rather than HTTP
Error: 1 problem in 1 formula detected
```
Also, we should try to remove `bottle :unneeded` here.

**Comment by nandahkrishna** [comment]
```
ad99eef61d3dcf6b29ef709e3374ab2d0a0002fbb664702dd2e6b05336f5a0ae  sqlmap--1.5.5.arm64_big_sur.bottle.tar.gz
61a72dadb8f2f77ecd3c6111e0cd35d75f7e12823e3a4aef254b5ae07678b3bb  sqlmap--1.5.5.big_sur.bottle.tar.gz
61a72dadb8f2f77ecd3c6111e0cd35d75f7e12823e3a4aef254b5ae07678b3bb  sqlmap--1.5.5.catalina.bottle.tar.gz
61a72dadb8f2f77ecd3c6111e0cd35d75f7e12823e3a4aef254b5ae07678b3bb  sqlmap--1.5.5.mojave.bottle.tar.gz
```

**Comment by alebcay** [comment]
```
--- sqlmap--1.5.5.arm64_big_sur.bottle.tar.gz
+++ sqlmap--1.5.5.big_sur.bottle.tar.gz
│   --- sqlmap--1.5.5.arm64_big_sur.bottle.tar
├── +++ sqlmap--1.5.5.big_sur.bottle.tar
│ ├── file list
│ │ @@ -252,29 +252,29 @@
│ │  -rw-r--r--   0        0        0   189608 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/common.py
│ │  -rw-r--r--   0        0        0     7496 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/compat.py
│ │  -rw-r--r--   0        0        0    12887 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/convert.py
│ │  -rw-r--r--   0        0        0      727 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/data.py
│ │  -rw-r--r--   0        0        0     6235 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/datatype.py
│ │  -rw-r--r--   0        0        0     2633 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/decorators.py
│ │  -rw-r--r--   0        0        0      556 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/defaults.py
│ │ --rw-r--r--   0        0        0    17381 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/dicts.py
│ │ +-rw-r--r--   0        0        0    17408 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/dicts.py
│ │  -rw-r--r--   0        0        0    28086 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/dump.py
│ │  -rw-r--r--   0        0        0    13858 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/enums.py
│ │  -rw-r--r--   0        0        0     1645 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/exception.py
│ │  -rw-r--r--   0        0        0    10051 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/gui.py
│ │  -rw-r--r--   0        0        0     5488 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/log.py
│ │  -rw-r--r--   0        0        0   103946 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/option.py
│ │  -rw-r--r--   0        0        0     7254 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/optiondict.py
│ │  -rw-r--r--   0        0        0     4977 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/patch.py
│ │  -rw-r--r--   0        0        0      884 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/profiling.py
│ │  -rw-r--r--   0        0        0     1800 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/readlineng.py
│ │  -rw-r--r--   0        0        0     4851 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/replication.py
│ │  -rw-r--r--   0        0        0     1875 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/revision.py
│ │  -rw-r--r--   0        0        0     1979 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/session.py
│ │ --rw-r--r--   0        0        0    45206 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/settings.py
│ │ +-rw-r--r--   0        0        0    45251 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/settings.py
│ │  -rw-r--r--   0        0        0     4946 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/shell.py
│ │  -rw-r--r--   0        0        0     5658 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/subprocessng.py
│ │  -rw-r--r--   0        0        0    32950 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/target.py
│ │  -rw-r--r--   0        0        0    16315 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/testing.py
│ │  -rw-r--r--   0        0        0     7802 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/threads.py
│ │  -rw-r--r--   0        0        0      987 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/unescaper.py
│ │  -rw-r--r--   0        0        0     7456 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/core/update.py
│ │ @@ -287,15 +287,15 @@
│ │  -rw-r--r--   0        0        0     2702 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/parse/handler.py
│ │  -rw-r--r--   0        0        0     1459 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/parse/headers.py
│ │  -rw-r--r--   0        0        0     3021 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/parse/html.py
│ │  -rw-r--r--   0        0        0     3577 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/parse/payloads.py
│ │  -rw-r--r--   0        0        0     1801 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/parse/sitemap.py
│ │  drwxr-xr-x   0        0        0        0 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/
│ │  -rw-r--r--   0        0        0      146 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/__init__.py
│ │ --rw-r--r--   0        0        0    18602 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/basic.py
│ │ +-rw-r--r--   0        0        0    18611 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/basic.py
│ │  -rw-r--r--   0        0        0     1383 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/basicauthhandler.py
│ │  -rw-r--r--   0        0        0     1565 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/chunkedhandler.py
│ │  -rw-r--r--   0        0        0     7625 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/comparison.py
│ │  -rw-r--r--   0        0        0    73462 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/connect.py
│ │  -rw-r--r--   0        0        0     3228 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/direct.py
│ │  -rw-r--r--   0        0        0     5972 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/dns.py
│ │  -rw-r--r--   0        0        0     5343 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/lib/request/httpshandler.py
│ │ @@ -723,15 +723,15 @@
│ │  -rw-r--r--   0        0        0    61489 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/identywaf/data.json
│ │  -rwxr-xr-x   0        0        0    25913 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/identywaf/identYwaf.py
│ │  drwxr-xr-x   0        0        0        0 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/keepalive/
│ │  -rw-r--r--   0        0        0      730 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/keepalive/__init__.py
│ │  -rw-r--r--   0        0        0    22806 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/keepalive/keepalive.py
│ │  drwxr-xr-x   0        0        0        0 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/magic/
│ │  -rw-r--r--   0        0        0        0 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/magic/__init__.py
│ │ --rw-r--r--   0        0        0     6741 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/magic/magic.py
│ │ +-rw-r--r--   0        0        0     6752 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/magic/magic.py
│ │  drwxr-xr-x   0        0        0        0 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/multipart/
│ │  -rw-r--r--   0        0        0        0 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/multipart/__init__.py
│ │  -rw-r--r--   0        0        0     4550 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/multipart/multipartpost.py
│ │  drwxr-xr-x   0        0        0        0 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/odict/
│ │  -rw-r--r--   0        0        0      156 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/odict/__init__.py
│ │  -rw-r--r--   0        0        0     4283 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/odict/ordereddict.py
│ │  drwxr-xr-x   0        0        0        0 2021-05-01 09:39:42.000000 sqlmap/1.5.5/libexec/thirdparty/prettyprint/
│ ├── sqlmap/1.5.5/libexec/lib/core/dicts.py
│ │ @@ -379,15 +379,15 @@
│ │  DUMP_DATA_PREPROCESS = {
│ │      DBMS.ORACLE: {"XMLTYPE": "(%s).getStringVal()"},  # Reference: https://www.tibcommunity.com/docs/DOC-3643
│ │      DBMS.MSSQL: {"IMAGE": "CONVERT(VARBINARY(MAX),%s)"},
│ │  }
│ │  
│ │  DEFAULT_DOC_ROOTS = {
│ │      OS.WINDOWS: ("C:/xampp/htdocs/", "C:/wamp/www/", "C:/Inetpub/wwwroot/"),
│ │ -    OS.LINUX: ("/var/www/", "/var/www/html", "/var/www/htdocs", "/usr/local/apache2/htdocs", "/usr/local/www/data", "/var/apache2/htdocs", "/var/www/nginx-default", "/srv/www/htdocs", "/usr/local/var/www")  # Reference: https://wiki.apache.org/httpd/DistrosDefaultLayout
│ │ +    OS.LINUX: ("/var/www/", "/var/www/html", "/var/www/htdocs", "@@HOMEBREW_PREFIX@@/apache2/htdocs", "@@HOMEBREW_PREFIX@@/www/data", "/var/apache2/htdocs", "/var/www/nginx-default", "/srv/www/htdocs", "@@HOMEBREW_PREFIX@@/var/www")  # Reference: https://wiki.apache.org/httpd/DistrosDefaultLayout
│ │  }
│ │  
│ │  PART_RUN_CONTENT_TYPES = {
│ │      "checkDbms": CONTENT_TYPE.TECHNIQUES,
│ │      "getFingerprint": CONTENT_TYPE.DBMS_FINGERPRINT,
│ │      "getBanner": CONTENT_TYPE.BANNER,
│ │      "getCurrentUser": CONTENT_TYPE.CURRENT_USER,
│ ├── sqlmap/1.5.5/libexec/lib/core/settings.py
│ │ @@ -886,15 +886,15 @@
│ │  NETSCAPE_FORMAT_HEADER_COOKIES = "# Netscape HTTP Cookie File."
│ │  
│ │  # Infixes used for automatic recognition of parameters carrying anti-CSRF tokens
│ │  CSRF_TOKEN_PARAMETER_INFIXES = ("csrf", "xsrf", "token")
│ │  
│ │  # Prefixes used in brute force search for web server document root
│ │  BRUTE_DOC_ROOT_PREFIXES = {
│ │ -    OS.LINUX: ("/var/www", "/usr/local/apache", "/usr/local/apache2", "/usr/local/www/apache22", "/usr/local/www/apache24", "/usr/local/httpd", "/var/www/nginx-default", "/srv/www", "/var/www/%TARGET%", "/var/www/vhosts/%TARGET%", "/var/www/virtual/%TARGET%", "/var/www/clients/vhosts/%TARGET%", "/var/www/clients/virtual/%TARGET%"),
│ │ +    OS.LINUX: ("/var/www", "@@HOMEBREW_PREFIX@@/apache", "@@HOMEBREW_PREFIX@@/apache2", "@@HOMEBREW_PREFIX@@/www/apache22", "@@HOMEBREW_PREFIX@@/www/apache24", "@@HOMEBREW_PREFIX@@/httpd", "/var/www/nginx-default", "/srv/www", "/var/www/%TARGET%", "/var/www/vhosts/%TARGET%", "/var/www/virtual/%TARGET%", "/var/www/clients/vhosts/%TARGET%", "/var/www/clients/virtual/%TARGET%"),
│ │      OS.WINDOWS: ("/xampp", "/Program Files/xampp", "/wamp", "/Program Files/wampp", "/Apache/Apache", "/apache", "/Program Files/Apache Group/Apache", "/Program Files/Apache Group/Apache2", "/Program Files/Apache Group/Apache2.2", "/Program Files/Apache Group/Apache2.4", "/Inetpub/wwwroot", "/Inetpub/wwwroot/%TARGET%", "/Inetpub/vhosts/%TARGET%")
│ │  }
│ │  
│ │  # Suffixes used in brute force search for web server document root
│ │  BRUTE_DOC_ROOT_SUFFIXES = ("", "html", "htdocs", "httpdocs", "php", "public", "src", "site", "build", "web", "www", "data", "sites/all", "www/build")
│ │  
│ │  # String used for marking target name inside used brute force web server document root
│ ├── sqlmap/1.5.5/libexec/lib/request/basic.py
│ │ @@ -294,15 +294,15 @@
│ │              return None
│ │  
│ │          try:
│ │              if contentEncoding == "deflate":
│ │                  data = io.BytesIO(zlib.decompress(page, -15))  # Reference: http://stackoverflow.com/questions/1089662/python-inflate-and-deflate-implementations
│ │              else:
│ │                  data = gzip.GzipFile("", "rb", 9, io.BytesIO(page))
│ │ -                size = struct.unpack("<l", page[-4:])[0]  # Reference: http://pydoc.org/get.cgi/usr/local/lib/python2.5/gzip.py
│ │ +                size = struct.unpack("<l", page[-4:])[0]  # Reference: http://pydoc.org/get.cgi@@HOMEBREW_PREFIX@@/lib/python2.5/gzip.py
│ │                  if size > MAX_CONNECTION_TOTAL_SIZE:
│ │                      raise Exception("size too large")
│ │  
│ │              page = data.read()
│ │          except Exception as ex:
│ │              if b"<html" not in page:  # in some cases, invalid "Content-Encoding" appears for plain HTML (should be ignored)
│ │                  errMsg = "detected invalid data for declared content "
│ ├── sqlmap/1.5.5/libexec/thirdparty/magic/magic.py
│ │ @@ -115,16 +115,16 @@
│ │              libmagic = ctypes.CDLL(dll)
│ │          except WindowsError:
│ │              pass
│ │  
│ │      if not libmagic or not libmagic._name:
│ │          import sys
│ │          platform_to_lib = {'darwin': ['/opt/local/lib/libmagic.dylib',
│ │ -                                      '/usr/local/lib/libmagic.dylib',
│ │ -                                      '/usr/local/Cellar/libmagic/5.10/lib/libmagic.dylib'],
│ │ +                                      '@@HOMEBREW_PREFIX@@/lib/libmagic.dylib',
│ │ +                                      '@@HOMEBREW_CELLAR@@/libmagic/5.10/lib/libmagic.dylib'],
│ │                             'win32':  ['magic1.dll']}
│ │          for dll in platform_to_lib.get(sys.platform, []):
│ │              try:
│ │                  libmagic = ctypes.CDLL(dll)
│ │              except OSError:
│ │                  pass
```

**Comment by carlocab** [comment]
Looks to be all safe for an
```ruby
inreplace file, "/usr/local", HOMEBREW_PREFIX
```

File: Formula/sqlmap.rb [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 
-  bottle :unneeded
-
   def install
     libexec.install Dir["*"]
+    
+    [libexec/"lib/core/dicts.py",
+     libexec/"lib/core/settings.py",
+     libexec/"lib/request/basic.py",
+     libexec/"thirdparty/magic/magic.py"].each { |f| inreplace f, "/usr/local", HOMEBREW_PREFIX }
```
Comment (carlocab) [inline Formula/sqlmap.rb:15]:
```suggestion

    files = [
      libexec/"lib/core/dicts.py",
      libexec/"lib/core/settings.py",
      libexec/"lib/request/basic.py",
      libexec/"thirdparty/magic/magic.py"
    ]
    inreplace files, "/usr/local", HOMEBREW_PREFIX
```

**Comment by carlocab** [comment]
```
❯ jq '.[].bottle.cellar' <bottles/*.bottle.json && rm -rf bottles
"any_skip_relocation"
"any_skip_relocation"
"any_skip_relocation"
"any_skip_relocation"
```

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**